packages/data-point/lib/accumulator/factory.js

```diff
@@ -8,11 +8,18 @@ const LocalsFactory = require('../locals/factory')
  */
 function Accumulator () {
   this.value = undefined
+  this.resolvedValue = null
   this.locals = undefined
   this.values = undefined
   this.reducer = undefined
   this.trace = false
   this.context = undefined
+  this.resolveTransformWith = function (value) {
+    return {
+      value: this.resolvedValue || value,
```

Comment:
Does `resolveTransformWith ` work if you want to resolve with a falsey value like `null` or `undefined`?

Reply:
You can set a falsetto value but there’s a bug. 😅

If a falsey value is set as the `resolvedValue` then it will be overwritten by any future function because the short circuit in `resolve/reducer-function#resolve` is checking the truthiness of `resolvedValue`.

I’ll add a fix for this and include tests to cover that scenario and the fix.

Great catch! 🙌

Reply:
Hopefully the new commit covers this case better 😄 

 https://github.com/ViacomInc/data-point/pull/41/commits/23917bbec96c78b1a5299b6fdfdc65c466d13a18